src/main/python/segment_collection/README.md

```diff
@@ -0,0 +1,40 @@
+### Segmenting collection
```

Comment:
Let's call this `passage_retrieval`?
So move everything to `src/main/python/passage_retrieval/`?